install_config/monitoring/configuring-alertmanager.adoc

```diff
@@ -0,0 +1,112 @@
+[[configuring-alertmanager]]
+== Configuring Alertmanager
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+The Alertmanager manages incoming alerts, including silencing, inhibition, aggregation and sending out notifications through methods such as email, PagerDuty and HipChat.
```

Comment:
s/aggregation and sending/aggregation, and sending
s/PagerDuty and HipChat/PagerDuty, and HipChat